eva_sub_cli/orchestrator.py

```diff
@@ -33,7 +34,7 @@
 DOCKER = 'docker'
 NATIVE = 'native'
 
-logger = logging_config.get_logger(__name__)
+logger = logging_config.get_logger(__name__, level=logging.DEBUG)
```

Comment:
This can be removed when https://github.com/EBIvariation/eva-common-pyutils/pull/69 is merged/released